readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://SSDoizha.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://SSDoizha.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="949" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/42610748-ca62-4827-bda0-b26d1edcf01f">